### PR TITLE
uniq: add page

### PR DESCRIPTION
**Pull request by muteness**
added page for uniq command

### CODE REVIEW

File: pages/common/uniq.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Display occurence of each line along with that line.
+
+`uniq -c {{file}}`
+
+- Display each duplicate line once.
+
+`uniq -d {{file}}`
+
+- Display unique lines.
```
Comment (igorshubovych) [inline pages/common/uniq.md]: I would rather change description to the following:
"Display only unique lines"

File: pages/common/uniq.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Display each line once.
+
+`uniq {{file}}`
+
+- Display occurence of each line along with that line.
+
+`uniq -c {{file}}`
+
+- Display each duplicate line once.
```
Comment (igorshubovych) [inline pages/common/uniq.md]: I would rather change description to the following:
"Display only duplicate lines"

**Comment by muteness** [comment]
fixed

**Comment by igorshubovych** [comment]
Please squash your commits by rebasing. Check out [contributing guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request) for instructions. Let us know if you have any issues.

Take a note, that your PR in the current state is removing sails.md, which has been recently merged. So please resolve this.

**Comment by muteness** [comment]
should I rebase to a0bc50f?
What should I do with sails.md? It seems that I should not simply add that back.

**Comment by igorshubovych** [comment]
Honestly this is beyond my Git experience. Maybe somebody more proficient with it can help.

None of your commits touches sails.md, but PR for some reason removes it. You can try different approaches. In worst case we will close this PR and you will open new PR with saved uniq.md.

**Comment by muteness** [comment]
OK. I will open a new PR.

**Comment by rubenvereecken** [comment]
Err I was about to say I could help you out, but good luck with next PR.

**Pull request closed without merging**